packages/aws-cdk-lib/aws-ec2/README.md

```diff
@@ -1052,6 +1052,16 @@ new ec2.VpcEndpointService(this, 'EndpointService', {
 });
 ```
 
+To include a service principal in the `allowedPrincipals`, there is a workaround where you can use a service principal string as input to the `ArnPrincipal` type. The resulting VPC endpoint will have an allowlisted principal of type `Service`, instead of `Arn` for that item in the list.
+```ts
```

Comment:
```suggestion
You can also include a service principal in the `allowedPrincipals` property by specifying it as a parameter to the  `ArnPrincipal` constructor.
The resulting VPC endpoint will have an allowlisted principal of type `Service`, instead of `Arn` for that item in the list.

```ts
```

Reply:
I'll make this change in the revision that includes the tests.